doc/sphinx/source/recipes/recipe_seaice_drift.rst

```diff
@@ -0,0 +1,47 @@
+.. _recipes_seaice_drift:
+
+Seaice drift
+============
+
+Overview
+--------
+[Please, fill this]
```

Comment:
Please, fill this.

Reply:
Forgot to paste the paper text :man_facepalming: